pkg/modules/pdfengines/routes.go

```diff
@@ -145,3 +145,57 @@ func convertRoute(engine gotenberg.PDFEngine) api.Route {
 		},
 	}
 }
+
+func encryptRoute(engine gotenberg.PDFEngine) api.Route {
+	return api.Route{
+		Method:      http.MethodPost,
+		Path:        "/forms/pdfengines/encrypt",
+		IsMultipart: true,
+		Handler: func(c echo.Context) error {
+			ctx := c.Get("context").(*api.Context)
+
+			// Let's get the data from the form and validate them.
+			var (
+				inputPaths  []string
+				ownerPassword string
+				userPassword string
+				keyLength int
+			)
+
+			err := ctx.FormData().
+				MandatoryPaths([]string{".pdf"}, &inputPaths).
+				MandatoryString("ownerPassword", &ownerPassword).
+				MandatoryString("userPassword", &userPassword).
+				Int("keyLength", &keyLength, 256).
```

Comment:
I think we have to validate if the value is either 40, 128 or 256.

Reply:
extracted into pkg/gotenberg/encryption.go. Hope thats ok